lib/winston-graylog2.js

```diff
@@ -53,6 +53,8 @@ function prepareMeta(meta) {
     });
   }
 
+  meta = _.merge(meta, this.staticMeta);
```

Comment:
Id rather add a parameter tot he function it's self: not been a method of the object is unclear why we'd have a this invocation here

@odino wdyt?